micro-docker/README.md

```diff
@@ -1,4 +1,53 @@
-- cd micro-docker
-- sudo docker build -t rackhd/micro .
-- sudo docker save rackhd/micro | xz -z > discovery.docker.tar.xz
-- copy discovery.docker.tar.xz to on-http/static/http/common
+# Micro Docker
+
+## RancherOS vmlinuz
```

Comment:
vmlinuz and initrd is not part of docker, it is easy to get confused to put description on them here.